src/js/pages/LoginPage.js

```diff
@@ -76,73 +76,171 @@ export default class LoginPage extends Component {
     }
   }
 
+  parseNormalBoldText(string) {
+    return (ParseText(string).map(text => (
```

Comment:
Can we please add a comment here that points at the markup-conversion utility class and explains the logic behind what's going on here? That will allow readers to better understand what precisely is happening here (it was unclear to me until I scrolled farther down in this changeset, and readers won't have the benefit of seeing all the associated changes in a single PR when they're looking at the code for the first time)

Reply:
Added comments